packages/vat-data/src/types.d.ts

```diff
@@ -23,24 +22,44 @@ type ActualBehavior<B> = {
   [Facet in keyof B]: FunctionsMinusContext<B[Facet]>;
 };
 
-type KindContext<S, B> = { state: S; facets: ActualBehavior<B> };
+type MultiKindContext<S, B> = { state: S; facets: ActualBehavior<B> };
```

Comment:
I find it weird that for single facet we name the helper `FunctionsMinusContext` but for multi, it's called `ActualBehavior`. In my mind it's `FacetFromBehavior` and `MultiFacetFromBehaviors` or something like that.

Reply:
That's out of scope of this change. Are you blocking this change on me renaming the extant types?